config/config.go

```diff
@@ -361,7 +447,10 @@ var defaultConf = Config{
 		SlowQueryFile:       "tidb-slow.log",
 		SlowThreshold:       logutil.DefaultSlowThreshold,
 		ExpensiveThreshold:  10000,
-		DisableErrorStack:   true,
+		DisableErrorStack:   nbUnset,
+		EnableErrorStack:    nbUnset, // If both options are unset, getDisableErrorStack() returns true
```

Comment:
What does `both options` mean?

Reply:
Both options refer to `DisableErrorStack` and `EnableErrorStack`, `DisableErrorStack` is kept for back compatibility, but to make it possible to detect what the user has put in the configuration file, we have to make both option unset by default, in which case `getDisableErrorStack` returns true.